app/src/main/res/values/styles.xml

```diff
@@ -202,10 +202,25 @@
     <item name="fontFamily">sans-serif-medium</item>
   </style>
 
-  <style name="Heading3" parent="TextView.Common1">
+  <style name="HeadingViewStart2" parent="TextView.Common1">
```

Comment:
Suggest inheriting from Heading2 and just defining the new property. Ditto for the other new properties.

Reply:
Done,